### PR TITLE
Mdb/landing page and docs

### PR DESCRIPTION
**Pull request by michaeldboyd**
Here is the combination of @MichaelEdwardBlack's landing page and my docs edits. Please don't merge until we're ready for launch

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,32 +1,57 @@
...
+  - navigation
   - toc
 ---
-# Introduction
+<div style="display: flex; flex-wrap: wrap;">
+  <div style="display: flex; flex-direction: column; width: 50%;">
+    <h1 style="font-weight: bold;">Welcome to the Docs</h1>
+    <h3>Here you'll find guides, resources, and references to build with Trinsic</h3>
+  </div>
+  <img src="/_static/images/person-welcome.png"/>
```
Comment (michaeldboyd) [inline docs/index.md]: image
I'd like to put this icon on the right 2/3rds of this splash screen. @MichaelEdwardBlack would you be will to add the magic css incantation to make that happen?

Comment (geel9) [reply docs/index.md]: I'll take care of it

**Comment by rileyphughes** [comment]
Landing page pixar:
- [x] Add [Looking for our Legacy Documentation](https://legacy.docs.trinsic.id/) at the very bottom of Home tab/landing page
- [x] Landing page Header: "Welcome to Trinsic's Docs 👋"
- [x] Landing page copy: "Build the future of identity with Trinsic, infrastructure for building amazing identity products using identity wallets and verifiable credentials."
- [x] Landing page box1: "CLI Tutorial | Dive into a sandbox CLI environment. | Get started"
- [x] Landing page box2: "Dashboard | Create your ecosystem and API key | Sign in"
- [x] Landing page box3: "API reference | Explore our server- and client-side API libraries. | View reference"
- [x] Position illustration on the right-hand side of the page for better aesthetic appeal

**Comment by rileyphughes** [comment]
Sidebar Pixar:
- [ ] Remove "Learn" as a section title on sidebar (see image)
	- Can't really be done
- [x] Change word "Overview" to "Introduction" (also see image)
image

**Comment by rileyphughes** [comment]
Intro page content pixar

- [x] Change intro paragraph to: "Trinsic is a cloud-native, web5-native infrastructure for building IDtech products across a variety of use cases, industries, and geographies. Products built on Trinsic inherit features such as user-centricity, interoperability, and decentralization out of the box. We strive to make these concepts easy to understand and use.
.
We’re glad you’re here. We’ve been focused on providing an exceptional developer experience for decentralized identity since our beginnings in 2019. Today, we power many of the most successful decentralized identity products. As you explore our documentation, we’d love your feedback through our Slack community, the chat bubble on the page, or [email](mailto:support@trinsic.id). And if you want to complain (or say nice things) to the CEO, feel free to email me (Riley at trinsic.id)."
 - [x] Remove the 3 subheadings & content (so there is nothing after the above paragraphs)

**Review by geel9 — APPROVED** [review]

**Pull request merged**